council/llm/__init__.py

```diff
@@ -43,30 +42,24 @@
 from .llm_function_with_prompt import LLMFunctionWithPrompt
 from .monitored_llm import MonitoredLLM
 
-from .chat_gpt_configuration import ChatGPTConfigurationBase
-from .openai_chat_completions_llm import OpenAIChatCompletionsModel
-from .openai_token_counter import OpenAITokenCounter
-
-from .azure_chat_gpt_configuration import AzureChatGPTConfiguration
-from .azure_llm import AzureLLM
-
-from .openai_chat_gpt_configuration import OpenAIChatGPTConfiguration
-from .openai_llm import OpenAILLM
-
-from .anthropic_llm_configuration import AnthropicLLMConfiguration
-from .anthropic_llm import AnthropicLLM
-
-from .gemini_llm_configuration import GeminiLLMConfiguration
-from .gemini_llm import GeminiLLM
-
-from .ollama_llm_configuration import OllamaLLMConfiguration
-from .ollama_llm import OllamaLLM
-
-from .groq_llm_configuration import GroqLLMConfiguration
-from .groq_llm import GroqLLM
+from .providers import (
```

Comment:
Don't import low-level `ChatGPTConfigurationBase`, `OpenAIChatCompletionsModel` and `OpenAITokenCounter`  here by analogy to not importing `AnthropicCompletionLLM`, `AnthropicMessagesLLM` from `anthropic`